tests/cc13d/input.dat

```diff
@@ -0,0 +1,25 @@
+memory 500 mb
+
+molecule water{
+0 1
+O
+H 1 R
+H 1 R 2 A
+
+R = 1.0
+A = 104.5
+}
+
+set {
+  reference RHF
+  basis cc-pVDZ
+  dertype first
```

Comment:
Why setting `dertype`? It really shouldn't ever be used by user, instead being set by fn in proc called by gradient. Is this superfluous? Also, fyi, I think we can stop setting memory in test cases unless significantly more or less than the default of 500mib is needed.